extensions/resource-deployment/src/services/azdataService.ts

```diff
@@ -38,7 +38,6 @@ export class AzdataService implements IAzdataService {
 			default:
 				throw new Error(`Unknown deployment type: ${deploymentType}`);
 		}
-		await this.ensureWorkingDirectoryExists();
```

Comment:
this is no longer needed becaue an initialized PlatformService will always the the storagePath ensured.